manifest_api.py

```diff
@@ -66,7 +66,12 @@
 def create_app():
     ''' Create a Flask app to be used/regestered by available blueprints. '''
     mainapp = flask.Flask('tm_manifesting', static_url_path='/static')
-    mainapp.config.update(manconfig)
+    #Need to have our custom config object from build_config.py, but want to keep
+    #flask's parameters as well.
+    flask_cfg = mainapp.config
```

Comment:
Line 71-73: most controversial change in this request...

Reply:
I guess I'm rusty on flask, because I don't understand the severity of the warning?

Reply:
The problem is: we used to call update() function on flask's config to add properties from our ManifestingConfiguration dictionary. However, I need GOLDEN_IMAGE to be class property. Plus, seemed to make more sense to carry around our ManifestingConfiguration config, instead of Flask's one.

Reply:
I don't know why GOLDEN_IMAGE needs to be class property because there's only ever one instance, but it probably won't hurt anything.